.buildkite/premerge.steps.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I tweaked this file to remove a bunch of boilerplate comments and change the step label to something that works a bit better with the GitHub status checks API (spaces in status names can make it unhappy sometimes).